### ISSUE TITLE
🧑‍💻: Add more Project with better visualizations 

### ISSUE THREAD

**Issue by yashksaini-coder** [UTSAVS26/PyVerse#860]
### Title

Add models for better visualizations

### Enhancement Aim

Add more better accuracy models, with better visual plotting

### Changes

Add Lightgbm, Xgboost

### Guidelines

- [X] I have read the guidelines
- [X] I have the link to my latest merged PR

### Full Name

Yash K. Saini

### Participant Role

GSSoC-Ext

**Comment by yashksaini-coder** [comment]
@UTSAVS26 can you assign this issue to me
